data/generate.py

```diff
@@ -0,0 +1,128 @@
+import requests
```

Comment:
Code is mostly generated. It took me 2 prompts and a few manual edits to adjust the chosen GraphQL field names.